### PR TITLE
Add help v2

### PR DESCRIPTION
**Pull request by hmanhduc2k**
Add skeletal code for various help function, modify existing help window and improve on the GUI to view Markdown content from UG.

### CODE REVIEW

File: src/main/java/seedu/academydirectory/ui/HelpWindow.java [hunk 1]
```diff
@@ -34,7 +43,33 @@
      */
     public HelpWindow(Stage root) {
         super(FXML, root);
-        helpMessage.setText(HELP_MESSAGE);
+        StringBuilder builder = new StringBuilder(HELP_MESSAGE);
+        MarkdownView markdownView = new MarkdownView();
+        try {
```
Comment (phuccuongngo99) [inline src/main/java/seedu/academydirectory/ui/HelpWindow.java:48]: Hmm, this try-catch block could be refactored into a small function so it looks cleaner :grin:

File: src/main/java/seedu/academydirectory/logic/commands/HelpCommand.java [hunk 2]
```diff
@@ -8,14 +10,50 @@
...
+        return String.format(MESSAGE_HELP_SUCCESS, syntax);
+    }
 
     @Override
     public CommandResult execute(Model model) {
         return new CommandResult(SHOWING_HELP_MESSAGE, true, false);
     }
+
+    @Override
+    public boolean equals(Object obj) {
```
Comment (phuccuongngo99) [inline src/main/java/seedu/academydirectory/logic/commands/HelpCommand.java:49]: Correct me if I'm wrong. `equals()` methods of Commands are used for testing purposes right?
Do u plan to update [HelpCommandTest.java](https://github.com/AY2122S1-CS2103T-T15-3/tp/blob/master/src/test/java/seedu/academydirectory/logic/commands/HelpCommandTest.java) in this PR?

Comment (phuccuongngo99) [reply src/main/java/seedu/academydirectory/logic/commands/HelpCommand.java:49]: Ah so you will leave the testing for Milestone v1.2b right?

Comment (hmanhduc2k) [reply src/main/java/seedu/academydirectory/logic/commands/HelpCommand.java:49]: Yep, I will
Or if possible I will start with the testing for next week PR

**Pull request merged**